2-mandatory/9-cheap-diner.js

```diff
@@ -19,58 +19,103 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+  if (mealArray.length === 0) {
+    return null;
```

Comment:
Is this returning the right thing? There is a specific answer we should be giving for if there is no food